homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
```

Comment:
I agree with the comment about the default implementation of send_commands. However, if we are keeping the `num_repeats` parameter as we were discussing below, I'm not sure if we'd want to fold `send_command` and `send_commands` into one service, unless we're fine with being able to repeat a list of commands as well. I don't actually see a problem with this, but I just hadn't thought of it originally.

Also, I'm slightly confused about the backwards compatible comment. These changes shouldn't break backward compatibility because the new parameters for the `send_command` service have defaults specified. And `send_commands` wouldn't break backwards compatibility because it's a new service.